### ISSUE TITLE
NY - Onboarding - Test HL7 with fake data

### ISSUE THREAD

**Issue by brick-green** [CDCgov/prime-reportstream#5626]
## Problem statement

The HL7 structure needs to be tested with NY. This con be completed in parallel with the work for the REST transport.

## Acceptance criteria

- [ ] NY accepts HL7 structure

## To do

- [x] Create HL7 files from CLI with fake data
- [x] received feedback
- [ ] implement settings changes as needed

**Comment by brick-green** [comment]
What value do you want in MSH-5?  ECLRS
What value do you want in MSH-6?  NYSDOH
Do you require us to put our own identifier and CLIA in MSH-4?  The CLIA should be ‘10DRPTSTRM’ unless you have your own.
Would you like to receive the AoE questions?  Not required
Do we need to suppress the QST flag in OBX-29? Not required
Are there any other fields that are required? For example, the ordering provider information is sometimes missing, and some states won’t accept messages without it.  We will process records if values are not populated.
Do you require a specific value in MSH-21-1 (Message profile ID)?  No

**Comment by brick-green** [comment]
Data sent and feedback received. NY registers all CLIA numbers so I proposed a centralized CLIA approach.

**Comment by sliu1000** [comment]
@BerniXiongA6 @greene858 - user sent response waiting for additional action, if any.


Good morning Tom,

We received your test message at 6:09pm and successfully forwarded it along to the ECLRS program. We can be available for additional testing.

 

Thank you,

 

T.J. Treharne
Health Informatics

**Comment by brick-green** [comment]
Closing as duplicate of 6545
